src/garage/tf/baselines/gaussian_cnn_baseline.py

```diff
@@ -27,6 +28,15 @@ def __init__(
             regressor_args=None,
             name='GaussianCNNBaseline',
     ):
+        assert isinstance(env_spec.observation_space, akro.Box) and \
```

Comment:
wait -- what if I get an `akro.Image`? is that not allowed?

Reply:
akro.Image is a subclass of akro.Box, so that'll work fine.

Reply:
ah, thanks for pointing that out.